docs/topics/mpp/multiplatform.md

```diff
@@ -80,34 +82,21 @@ Multiplatform as well.
 
 The [Ktor framework](https://ktor.io/) is suitable for building asynchronous servers and clients in connected systems.
 
-## What's next?
+## How to get started
 
-New to Kotlin? Visit [Getting started with Kotlin](getting-started.md).
+New to Kotlin? Take a look at [Getting started with Kotlin](getting-started.md).
```

Comment:
Maybe we should drop the link to the common Kotlin Get started? For the Kotlin Multiplatform page, I'd expect _How to get started_ to be about Multiplatform and KMM only

Reply:
Don't think that it's a good idea. We could discuss this later

Reply:
Substituted with the note